src/domains/movie.js

```diff
@@ -0,0 +1,80 @@
+const prisma = require('../utils/prisma')
+
+// GET MOVIE LIST
+const getMovieListDb = async () => await prisma.movie.findMany({
+    include: { screenings: true }
+})
+
+// GREATER THAN AND LESS THAN RT
+const getMovieListGtLtDb = async (runtimeLt, runtimeGt) => await prisma.movie.findMany({
+    where: {
+        OR: [
```

Comment:
Isn't it an AND in the case that both a "greater than" and "less than" is supplied, implying a range?

Reply:
The logic should be correct, I couldn't use AND here because that would imply the provided ranges would both need to be met to return a result. In a case where say I wanted all movies less than 100 mins and more than 150 mins long, using AND would look for movies that fit both those criteria, which is obviously impossible lol.